papaparse.js

```diff
@@ -193,7 +193,7 @@
 	function CsvToJson(_input, _config)
 	{
 		_config = _config || {};
-		_config.dynamicTyping = _config.dynamicTyping || false;
+		_config.dynamicTyping = new DynamicTypingConfig(_config.dynamicTyping);
```

Comment:
I doubt if we should test if the dinamicTyping has the `isIncluded` function, so the user can build it's own `isIncluded` function to return if some field is included or not, in case only and except is not enough.